spec/controllers/comments_controller_spec.rb

```diff
@@ -1030,6 +2304,48 @@
         post :create, params: { work_id: work.id, comment: anon_comment_attributes }
         it_redirects_to_with_error(work_path(work), edit_error_message)
       end
+
+      context "PUT #freeze" do
+        # TODO: Should this be more like check_permission_to_delete and use access_denied?
```

Comment:
I think this way is fine. `access_denied` currently doesn't let us use `redirect_back`.

Reply:
Feel free to remove these TODOs unless you'd like a consistency cleanup later.